src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/AbstractDataFlowAnalysisContext.cs

```diff
@@ -99,57 +98,6 @@ public abstract class AbstractDataFlowAnalysisContext<TAnalysisData, TAnalysisCo
             ValueContentAnalysisResult? valueContentAnalysisResultOpt,
             InterproceduralAnalysisData<TAnalysisData, TAnalysisContext, TAbstractAnalysisValue>? interproceduralAnalysisData);
 
-        public ControlFlowGraph? GetLocalFunctionControlFlowGraph(IMethodSymbol localFunction)
```

Comment:
Hmm I am not sure this can be removed. Hopefully, there are unit tests to guard it.

Reply:
I am having quite a lot of issues with the unit tests lately. Either they don't run (there was an unexpected error) or appear green but are actually broken. I have tried cleaning the repo but no change.

The behavior is correct when running `Test.cmd`.

Are you also facing issues?